src/py42/sdk/queries/fileevents/file_event_query.py

```diff
@@ -36,11 +54,47 @@ def create_not_exists_filter_group(term):
     return create_filter_group(filter_list, u"AND")
 
 
+def create_greater_than_filter_group(term, value):
+    filter_list = [create_query_filter(term, u"GREATER_THAN", value)]
+    return create_filter_group(filter_list, u"AND")
+
+
+def create_less_than_filter_group(term, value):
+    filter_list = [create_query_filter(term, u"LESS_THAN", value)]
+    return create_filter_group(filter_list, u"AND")
+
+
 class FileEventFilterStringField(QueryFilterStringField):
+    """Helper class for creating filters with the ``EXISTS``/``NOT_EXISTS`` filter clauses."""
+
     @classmethod
     def exists(cls):
+        """Returns a FilterGroup to find events where filter data exists."""
```

Comment:
A couple more FilterGroups here you could put in class links.... I know there are a lot of them, but you never know which part of the docs the user will read and when.

Reply:
Grepped for all instances of this, so they should all be linked now.